lib/travis/build/script/c.rb

```diff
@@ -3,7 +3,7 @@ module Build
     class Script
       class C < Script
         DEFAULTS = {
-          compiler: 'gcc'
+          compiler: ''
```

Comment:
What I wanted to achieve was to use different default compilers for Linux (gcc) and Mac (clang). It could be due to my limited knowledge of Ruby and/or Travis, but I failed to setup an if with `:os` at this point, so I kept the default blank and overwrite it later.